TinCan/ActivityDefinition.cs

```diff
@@ -13,7 +13,16 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */
+
+/*
+ *  Modified by Paul Carpenter 2015
+ *  
+ *  Includes support for Activity Interactions
+ *  
+ */
```

Comment:
I'll check with Oliver about this. It may be that this was just intended to be an internal note so they know what they changed from the library, which becomes unnecessary if/when this is merged anyway.